int8 casualty_id
float32 heart_rate
float32 respiratory_rate
